feat: add page descriptions and update titleType (#340)
carbon-design-system/carbon-website: src/pages/contributions/bugs-and-requests.mdx

---
title: Bugs and requests

> description: We use Github to track our bugs. If you have a bug to report or wish to request a new feature, please check our existing issues before opening a new one.

Reviewer: ```suggestion
description: Carbon tracks bugs and issues with GitHub. If you have a bug to report or wish to request a new feature, please check our existing issues before opening a new one.
```
